gems/aws-sdk-core/lib/aws-sdk-core/param_validator.rb

```diff
@@ -152,7 +152,7 @@ def shape(ref, value, errors, context)
       when MapShape then map(ref, value, errors, context)
       when DocumentShape then document(ref, value, errors, context)
       when StringShape
-        unless value.is_a?(String)
+        unless value.is_a?(String) || ref['jsonvalue']
```

Comment:
I don't think this one is needed. The jsonvalue converter should run before this.

Reply:
Yup - removed.